patch/0012-drivers-hwmon-support-LTC3888.patch

```diff
@@ -0,0 +1,116 @@
+From e32f1ade04454f2c3df1599c61fbd5e562ab7e05 Mon Sep 17 00:00:00 2001
+From: Ashwin H <Ashwin.H@amd.com>
+Date: Tue, 27 Jun 2023 06:32:53 +0000
+Subject: [PATCH 12/14] drivers/hwmon: Adding support LTC3888
+
+This patch adds support for the LTC3888 VRM.
```

Comment:
Please elaborate, and document the datasheet you used.

Reply:
I think we don't specify datasheet details in commit message for a new chip as i see in the commit history.

However datasheet is here https://www.analog.com/en/products/ltc3888.html